examples/hybrid/types.jl

```diff
@@ -69,6 +69,10 @@ function microphysics_model(parsed_args)
     end
 end
 
+function anelastic_dycore(parsed_args)
+    anelastic_dycore = parsed_args["anelastic_dycore"]
+end
+
```

Comment:
```suggestion
```

Reply:
sorry this is part of #668 , I am just prototyping here